shpc/settings.yml

```diff
@@ -50,6 +50,12 @@ docker_shell: /bin/sh
 # shell for test.sh file
 test_shell: /bin/bash
 
+# for container aliases, use wrapper shellscripts instead of shell aliases, defaults to false
+wrapper_scripts: false
+
+# shell for wrapper shellscripts
+wrapper_shell: /bin/bash
```

Comment:
Can we please put this under the other shells so they are grouped together?

Reply:
addressed in commit https://github.com/marcodelapierre/singularity-hpc/commit/028a1d6c1e382e36f5b9180f56d4ef6e5715eb10